Makefile

```diff
@@ -39,12 +39,8 @@ goreleaser:
 build: goreleaser
 	goreleaser build --snapshot --rm-dist
 
-export IMAGE_TAG=0.0.0-development
-
-build/docker:
-	docker build --build-arg TELEMETRY_WRITE_KEY=${TELEMETRY_WRITE_KEY} . -t infrahq/infra:$(IMAGE_TAG)
-
-export OKTA_SECRET=infra-okta
+dev/docker:
```

Comment:
Is this supposed to be in this PR? Do you need to rebase?

Reply:
O.o i just rebased not 100% sure why it is in here .. let me double check

Reply:
oh got it! i forgot to push the rebase changes to the `layout-updates` branch